chain/watcher_test.go

```diff
@@ -181,7 +187,7 @@ func collectBlockHeaders(n lens.API, ts *types.TipSet) (blockHeaderList, error)
 	blocks := ts.Blocks()
 
 	for _, bh := range ts.Blocks() {
-		if bh.Height == 0 {
+		if bh.Height < 2 {
```

Comment:
This change ensures the list of blocks returned by `collectBlockHeaders` never returns the genesis block which we don't process.

Reply:
Due to the below line (188) adding bh.Parents to the blocks list, parent of 1 is 0 and we don't want 0 (genesis) in the list since we can't process it.